### PR TITLE
2634: Don't show last update for empty category with embedded offers

### PR DESCRIPTION
**Pull request by Njivaniaina2407**
### Short description

Showing a last update for an empty category with only embedded offers is pretty confusing, because the list of the offers was not updated last time on that dates.

### Proposed changes

Set a condition before displaying the last update

### Resolved issues

Fixes: #2634 (web only)

---

### CODE REVIEW

File: web/src/components/Page.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ const Page = ({
   thumbnailSrcSet,
   content,
   lastUpdate,
-  showLastUpdateText = true,
+  showLastUpdateText = false,
```
Comment (steffenkleinle) [inline web/src/components/Page.tsx:39]: I think you shouldn't just change this. This changes behavior in a lot of other use cases. Please revert this and instead pass `false` as a prop in the specific case you don't want to show it.

File: web/src/components/LastUpdateInfo.tsx [hunk 2]
```diff
@@ -26,7 +26,7 @@ export const LastUpdateInfo = ({
   const { i18n, t } = useTranslation('common')
   return (
     <TimeStamp className={className}>
-      {withText && t('lastUpdate')} {lastUpdate.setLocale(i18n.language).toFormat(format)}
+      {withText && t('lastUpdate')} {withText && lastUpdate.setLocale(i18n.language).toFormat(format)}
```
Comment (steffenkleinle) [inline web/src/components/LastUpdateInfo.tsx:29]: This changes the behavior of the prop. The prop is just there to decide whether to just display a date or a date and the text `last update:`.  A better option would be to just not render the whole component if you don't want it to show at all.

**Review by steffenkleinle — CHANGES_REQUESTED** [review]

**Review by steffenkleinle — CHANGES_REQUESTED** [review]
Looking good.

Native is missing though, I can take over :)

**Review by steffenkleinle — APPROVED** [review]
Looks good! Lets do native in a separate PR :)

**Review by ztefanie — APPROVED** [review]
Tested on Chrome works nice 🎉

**Pull request merged**